awd/dynamodb.py

```diff
@@ -19,15 +21,24 @@ def __init__(self, region):
 
     def add_doi_item_to_database(self, doi_table, doi):
         """Add DOI item to database table."""
-        response = self.client.put_item(
-            TableName=doi_table,
-            Item={
-                "doi": {"S": doi},
-                "status": {"S": str(Status.PROCESSING.value)},
-                "attempts": {"S": "0"},
-            },
-        )
-        return response
+        try:
+            response = self.client.put_item(
+                TableName=doi_table,
+                Item={
+                    "doi": {"S": doi},
+                    "status": {"S": str(Status.PROCESSING.value)},
+                    "attempts": {"S": "0"},
+                    "last_modified": {
+                        "S": datetime.now().strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
Since this (%Y-%m-%d %H:%M:%S) is used multiple times, it will be helpful to declare as a reusable string to avoid error.